Sources/Testing/ExitTests/ExitTest.swift

```diff
@@ -815,5 +911,62 @@ extension ExitTest {
       issueCopy.record()
     }
   }
+
+  /// Decode this exit test's captured values and update its ``capturedValues``
+  /// property.
+  ///
+  /// - Throws: If a captured value could not be decoded.
+  ///
+  /// This function should only be used when the process was started via the
+  /// `__swiftPMEntryPoint()` function. The effect of using it under other
+  /// configurations is undefined.
+  private mutating func _decodeCapturedValuesForEntryPoint() throws {
+    // Read the content of the captured values stream provided by the parent
+    // process above.
+    guard let fileHandle = Self._makeFileHandle(forEnvironmentVariableNamed: "SWT_EXPERIMENTAL_CAPTURED_VALUES", mode: "rb") else {
+      return
+    }
+    let capturedValuesJSON = try fileHandle.readToEnd()
+    let capturedValuesJSONLines = capturedValuesJSON.split(whereSeparator: \.isASCIINewline)
+    assert(capturedValues.count == capturedValuesJSONLines.count, "Expected to decode \(capturedValues.count) captured value(s) for the current exit test, but received \(capturedValuesJSONLines.count). Please file a bug report at https://github.com/swiftlang/swift-testing/issues/new")
```

Comment:
Given this is an `assert` (and thus disabled in optimized/release builds), the bug report link seems less important here. Or should it be `precondition`, or something stronger?

Reply:
Realistically, we can't actually hit this failure, but I do like sanity-checking from time to time. Hence it's just an assert.

Reply:
Sounds good, but then can we remove the unnecessary bug report link?